test/README.md

```diff
@@ -79,6 +79,21 @@ a "nop" 90 # Assembly is correct
 dB "nopppp" 90 # Disassembly test is broken
 ```
 
+#### IL
+
+To also test lifting an instruction to RzIL, you can append the readable IL
+representation like so:
+```
+d "inc ptr" 3e 0 set(v:ptr, x:add(x:var(v:ptr), y:bitv(bits:0x0000000000000001, len:64)))
```

Comment:
What about putting IL part into the quotes instead? @ret2libc @wargio @kazarmy what is your opinion?

Reply:
Right now it just parses until the end, so no quotes are necessary.

Reply:
Yes, I understood that from the code. I am just wondering if quotes are better. But I don't have a strong opinion about this, to be honest. Can be improved later, if necessary.

Reply:
I think we should rewrite the entire asm test syntax at some point, probably when `RzArch` exists, to also account for other op properties, so I wouldn't spend too much time on this right now.